docs/Release_notes.md

```diff
@@ -1,5 +1,139 @@
 CARMA Platform Release Notes
 ----------------------------
+
+Skyline Release, Version 3.1.0, released 21 October 2019 
```

Comment:
Since this is a DRAFT release notes at this point, it's okay to put TBDs in it.  For this release, it's pretty clear it will end up being called 3.1.0, but let's not start the habit of associating codenames to version numbers this early in the process - sometimes they will jump around. Also, let's not assign a date yet, since we don't know what testing will find.

Reply:
The release notes is always going to be one of those funny ones that will probably be changing at the last minute before merging into master.  So I don't mind having it merged into the release branch early, it just needs to be clear that it is a draft (labeled with codeword and not with version number).  If it's merged it may be a bit easier to evaluate the whole release package by just checking out the release branch and you can see everything in one place.